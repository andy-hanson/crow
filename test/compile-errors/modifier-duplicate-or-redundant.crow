main void()
	()

nat64 builtin data

e extern mut

ee extern extern

en enum data

f flags data

r record data

u union data
	r

uv union by-val
	r

dd extern data, data

mm record mut, mut

ss record shared, shared

rr record by-ref, by-ref

vv record by-val, by-val
