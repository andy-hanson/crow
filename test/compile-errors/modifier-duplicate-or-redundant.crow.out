exit code: 0
stderr:
test/compile-errors/modifier-duplicate-or-redundant.crow 4:1-4:2 'bare' is redundant given 'extern'.
test/compile-errors/modifier-duplicate-or-redundant.crow 4:16-4:20 Redundant 'bare'.
test/compile-errors/modifier-duplicate-or-redundant.crow 4:34-4:40 Redundant 'extern'.
test/compile-errors/modifier-duplicate-or-redundant.crow 6:15-6:19 A builtin type is already 'data' by default.
test/compile-errors/modifier-duplicate-or-redundant.crow 8:10-8:13 An extern type is already 'mut' by default.
test/compile-errors/modifier-duplicate-or-redundant.crow 10:11-10:17 An extern type is already 'extern' by default.
test/compile-errors/modifier-duplicate-or-redundant.crow 12:9-12:13 An enum type is already 'data' by default.
test/compile-errors/modifier-duplicate-or-redundant.crow 14:9-14:13 A flags type is already 'data' by default.
test/compile-errors/modifier-duplicate-or-redundant.crow 16:10-16:14 A record type is already 'data' by default.
test/compile-errors/modifier-duplicate-or-redundant.crow 18:9-18:13 A union type is already 'data' by default.
test/compile-errors/modifier-duplicate-or-redundant.crow 21:10-21:16 A union type is already 'by-val' by default.
test/compile-errors/modifier-duplicate-or-redundant.crow 24:17-24:21 Redundant 'data'.
test/compile-errors/modifier-duplicate-or-redundant.crow 26:16-26:19 Redundant 'mut'.
test/compile-errors/modifier-duplicate-or-redundant.crow 28:19-28:25 Redundant 'shared'.
test/compile-errors/modifier-duplicate-or-redundant.crow 30:19-30:25 Redundant 'by-ref'.
test/compile-errors/modifier-duplicate-or-redundant.crow 32:19-32:25 Redundant 'by-val'.
test/compile-errors/modifier-duplicate-or-redundant.crow 34:25-34:33 Redundant 'extern'.
test/compile-errors/modifier-duplicate-or-redundant.crow 36:20-36:27 Redundant 'builtin'.
